clerk/types/querier.go

```diff
@@ -14,3 +15,14 @@ type QueryRecordParams struct {
 func NewQueryRecordParams(recordID uint64) QueryRecordParams {
 	return QueryRecordParams{RecordID: recordID}
 }
+
+// QueryRecordListParams defines the params for querying accounts.
```

Comment:
How about we abstract this out and use the same struct for `checkpoint/list`, `span/list` and `clerk/list`

Reply:
Fixed.